modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/TaskUtil.java

```diff
@@ -223,51 +222,48 @@ public static Task toTask(
 	private static Map<String, String> _createMap(
 		Document document, String fieldName) {
 
-		return Stream.of(
-			document.getFields()
-		).map(
-			Map::entrySet
-		).flatMap(
-			Collection::stream
-		).filter(
-			entry ->
-				StringUtil.startsWith(
-					entry.getKey(), fieldName + StringPool.UNDERLINE) &&
-				!StringUtil.endsWith(entry.getKey(), "_sortable")
-		).collect(
-			Collectors.toMap(
-				entry -> _toLanguageTag(
+		Map<String, String> map = new HashMap<>();
+
+		Map<String, Field> fields = document.getFields();
+
+		for (Map.Entry<String, Field> entry : fields.entrySet()) {
+			String key = entry.getKey();
+
+			if (StringUtil.startsWith(key, fieldName + StringPool.UNDERLINE) &&
+				!StringUtil.endsWith(key, "_sortable")) {
+
+				Field field = entry.getValue();
+
+				map.put(
 					StringUtil.removeSubstring(
-						entry.getKey(), fieldName + StringPool.UNDERLINE)),
-				entry -> {
-					Field field = entry.getValue();
+						key, fieldName + StringPool.UNDERLINE),
+					String.valueOf(field.getValue()));
+			}
+		}
 
-					return String.valueOf(field.getValue());
-				})
-		);
+		return map;
 	}
 
 	private static Map<String, String> _createMap(
 		Map<String, Object> sourcesMap, String fieldName) {
 
-		return Stream.of(
-			sourcesMap
-		).map(
-			Map::entrySet
-		).flatMap(
-			Collection::stream
-		).filter(
-			entry ->
-				StringUtil.startsWith(
+		HashMapBuilder.HashMapWrapper<String, String> hashMapWrapper =
+			HashMapBuilder.create(sourcesMap.size());
+
+		for (Map.Entry<String, Object> entry : sourcesMap.entrySet()) {
+			if (StringUtil.startsWith(
 					entry.getKey(), fieldName + StringPool.UNDERLINE) &&
-				!StringUtil.endsWith(entry.getKey(), "_sortable")
-		).collect(
-			Collectors.toMap(
-				entry -> _toLanguageTag(
-					StringUtil.removeSubstring(
-						entry.getKey(), fieldName + StringPool.UNDERLINE)),
-				entry -> GetterUtil.getString(entry.getValue()))
-		);
+				!StringUtil.endsWith(entry.getKey(), "_sortable")) {
+
+				hashMapWrapper.put(
```

Comment:
@slnn Missed this one for https://github.com/liferay-core-infra/liferay-portal/pull/4006#pullrequestreview-1360692502